tools/migration/internal/repo_fs_wrangler.go

```diff
@@ -3,12 +3,14 @@ package internal
 import (
 	"os"
 	"strings"
+
+	rcopy "github.com/otiai10/copy"
 )
 
 // RepoFSWrangler manages filesystem operations and figures out what the correct paths
 // are for everything.
 type RepoFSWrangler struct {
-	oldRepoPath, newRepoPath string
+	oldVersion, oldRepoPath, newRepoPath string
```

Comment:
oldVersion is used for renaming/archiving the old repo.

Reply:
>One unspoken design decision is whether the wrangler should logically be re-usable, or tightly coupled to a specific migration process – I hadn't seen this decision before, am now angling for the former, but can be convinced the latter is better.

I think the wrangler code ought to be reusable; otherwise there will be a lot of needlessly duplicate code.

Reply:
> But can't it read it out of the repo? What happens if they conflict? Is it checked?
Do we want the wrangler to try to parse anything in the repo?  I figured that checking would happen outside of the wrangler.